element_animal/genotyping.py

```diff
@@ -6,14 +6,15 @@
 from . import subject
 
 schema = dj.schema()
+_linking_module = None
 
 
 def activate(
-    genotyping_schema_name,
-    subject_schema_name=None,
-    create_schema=True,
-    create_tables=True,
-    linking_module=None,
+    genotyping_schema_name: str,
+    subject_schema_name: str = None,
+    create_schema: bool = True,
+    create_tables: bool = True,
+    linking_module: str = None,
```

Comment:
Why `str`? `linking_module` is not of type `str`, it's of type `module` (or some sort of dot-map object). I'd suggest not to specify a type for it if not sure.

Reply:
That's a good point. I was following [this](https://github.com/datajoint/element-calcium-imaging/blob/main/element_calcium_imaging/scan.py#L22) example. The type may be wrong there.